web/packages/teleport/src/components/TextSelectCopy/TextSelectCopyMulti.tsx

```diff
@@ -55,12 +70,15 @@ export function TextSelectCopyMulti({ lines, bash = true }: Props) {
       pt={2}
       pr={7}
       borderRadius={2}
+      maxHeight={maxHeight}
```

Comment:
`maxHeight` added to enable scroller for larger content. This is useful to display certificate content in SAML app configuration flow.

<img width="782" alt="Screenshot 2024-01-24 at 7 04 04 PM" src="https://github.com/gravitational/teleport/assets/10177602/f557b331-eb69-42ef-9b59-3687fdd33b4a">